src/coreclr/gc/gcconfig.h

```diff
@@ -140,7 +140,8 @@ class GCConfigStringHolder
     STRING_CONFIG(GCName,                    "GCName",                    "System.GC.Name",                                        "Specifies the name of the standalone GC implementation.")                                \
     STRING_CONFIG(GCPath,                    "GCPath",                    "System.GC.Path",                                        "Specifies the path of the standalone GC implementation.")                                \
     INT_CONFIG   (GCSpinCountUnit,           "GCSpinCountUnit",           NULL,                                0,                  "Specifies the spin count unit used by the GC.")                                          \
-    INT_CONFIG   (GCDynamicAdaptationMode,   "GCDynamicAdaptationMode",   "System.GC.DynamicAdaptationMode",   0,                  "Enable the GC to dynamically adapt to application sizes.")
+    INT_CONFIG   (GCDynamicAdaptationMode,   "GCDynamicAdaptationMode",   "System.GC.DynamicAdaptationMode",   0,                  "Enable the GC to dynamically adapt to application sizes.")                               \
+    INT_CONFIG   (GCDTargetTCP,              "GCDTargetTCP",              "System.GC.DTargetTCP",              0,                  "Specifies the target tcp for DATAS")
```

Comment:
Since this does seem like a public property (by virtue of the public key) - probably worth adding a bit more details such as describing what TCP is, what the implications are of setting a high value if DATAS is enabled and any other nuances of using this.

Reply:
I think the name might change which was why I haven't put in much effort to document this. this can go out in preview5 so we can use the preview build to do some testing with partner teams. but I think the name needs more thoughts (its meant to mean "dynamic target tcp".